src/pathfora.js

```diff
@@ -88,7 +88,8 @@
         message: 'Message',
         company: 'Company',
         phone: 'Phone Number',
-        country: 'Country'
+        country: 'Country',
+        referrer: 'Referrer'
```

Comment:
this was a bad place to add this comment, to be clear i was speaking to the actually key not just the label. the label should definitely be "Referral Email" or something to that effect.